<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiniAHCI Test Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_1">
      <property name="title">
       <string>Available devices</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="deviceBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="portBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="refreshButton">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Operations</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="idButton">
         <property name="text">
          <string>Device ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="recalibrateButton">
         <property name="text">
          <string>Recalibrate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sleepButton">
         <property name="text">
          <string>Sleep</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sResetButton">
         <property name="text">
          <string>Soft Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="hResetButton">
         <property name="text">
          <string>Hard Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="smartButton">
         <property name="text">
          <string>SMART</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="readTestButton">
         <property name="text">
          <string>Read Test</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
